### ISSUE TITLE
[enhancement] plugin should deal with nested fields better

### ISSUE THREAD

**Issue by blysik** [logstash-plugins/logstash-output-graphite#13]
If I have a nested field like:

```
metrics => {
  requests => 1020
  foo => 984894
  bar => 323
}
```

Graphite output should walk the nested field, and DTRT in sending it to graphite.  In my example, it would do:

metrics.request 1020 timestamp
metrics.foo 984894 timestamp
metrics.bar 323 timestamp

**Comment by andrewvc** [comment]
@blysik is it the case that your expectation is using the `graphite` output like so:

```
graphite {
  fields_are_metrics => ['metrics'] 
}
```

should cause it to perform the transformation you're thinking of?

**Comment by blysik** [comment]
Yes!  That's exactly my expectation.  (Or some regex similar to that.)

**Comment by andrewvc** [comment]
@blysik great, we'll get that released in the next day or two. It'll work with either `metrics` or `fields_are_metrics`, your choice!

**Comment by blysik** [comment]
Awesome!  Look forward to trying it out.

**Comment by andrewvc** [comment]
Released in version 1.0.1 . Enjoy @blysik !
